exercise_01/transformStringToCollection.js

```diff
@@ -12,3 +12,21 @@
  * output
  *  - array
  */
+/**
+ * If received value is not a string or it is an empty string, then call onError callback
+   Otherwise, call onSuccess function
+ * @param {any} value
+ * @param {Function} onError
+ * @param {Function} onSuccess
+ * @returns array
+ */
+
+const transformStringToArray = function (value, onError, onSuccess) {
```

Comment:
Validations and error handling must be managed as first case.

It's not necessary creating a constant for this. You can directly invoke callback and return its value

```suggestion
const transformStringToArray = function (value, onError, onSuccess) {
  if (typeof value !== "string" && value === "") {
    return onError();
  }
  return onSuccess(value);
};
```

FIX: apply validation and error handling as first step